tls/s2n_x509_validator.c

```diff
@@ -549,3 +555,71 @@ s2n_cert_validation_code s2n_x509_validator_validate_cert_stapled_ocsp_response(
     return ret_val;
 #endif /* S2N_OCSP_STAPLING_SUPPORTED */
 }
+
+DEFINE_POINTER_CLEANUP_FUNC(STACK_OF(X509)*, wipe_cert_chain);
+
+S2N_RESULT s2n_x509_validator_validate_certificate_signatures(struct s2n_connection *conn, struct s2n_x509_validator *validator,
+                                                                             s2n_cert_validation_code *validation_code)
+{
+    ENSURE_REF(conn);
+    ENSURE_REF(validator);
+
+    if (conn->config->security_policy->certificate_signature_preferences == NULL) {
+        *validation_code = S2N_CERT_OK;
+        return S2N_RESULT_OK;
+    }
```

Comment:
What if there's an override policy on the connection? You need to use s2n_connection_get_security_policy to retrieve the policy.

Reply:
Also, do you have a test for this case? It's a pretty important case.